gha/bin/create_qa_execute.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3.10
+from os import environ
+from typing import NoReturn
+
+from gha.github.baseprcommand import BasePrCommand
+from gha.jira.basejiracommand import BaseJiraCommand
+from gha.jira.util import JIRA_ISSUE_RE, convert_description_to_jira
+from jira.resources import Issue
+
+PROJECT_KEY = environ.get("JIRA_PROJECT") or None
+QA_USER = environ.get("QA_USERNAME") or None
+ISSUE_TYPE = 'GBL.QA-Execute'
+
+
+class CreateQAExecute(BasePrCommand, BaseJiraCommand):
+    """
+    Create new Jira issue of QA.Execute type after pr merged.
+    """
+    issue: Issue
+
+    def handle(self) -> NoReturn:
+        title = self.pr.title
+        match = JIRA_ISSUE_RE.search(title)
+        if not match:
+            raise RuntimeError(f'Issue key is not found in "{title}"')
+        issue_key = match.group(0)
+        project_key = match.group(1)
+        self.log.info(f'Found issue key "{issue_key}" in project {project_key}')
+        if project_key != PROJECT_KEY:
+            self.log.info(f'The issue is not in {project_key} project, QA-Execute issue will not be created')
+        else:
+            qa_execute_issue = self.create_qa_execute(project_key=project_key,
+                                                      issue_key=issue_key,
+                                                      issuetype=ISSUE_TYPE)
```

Comment:
Мне всегда очень странны люди, которые игнорируют рядом лежащий код в отношении форматирования.

```suggestion
            qa_execute_issue = self.create_qa_execute(
                project_key=project_key,
                issue_key=issue_key,
                issuetype=ISSUE_TYPE
            )
```